src/components/SpritePane/SpriteCard.tsx

```diff
@@ -13,21 +13,50 @@ type SpriteCardProps = {
 export function SpriteCard({ target }: SpriteCardProps) {
     const [editingTarget, setEditingTarget] = useEditingTarget();
 
-    
+    let inputName = ''
+
+    // Upon a name change, re-render the sprite's card in a local variable.
+    // useEffect(() => {
+    //     if ((target?.sprite?.name).length > 16) {
+    //         inputName = (target?.sprite?.name).substring(0, 13) + '...'
+    //     }
+    //     else {
+    //         inputName = target?.sprite?.name
+    //     }
+
+    // }, [target?.sprite?.name])
+
     const onClick = () => {
         setEditingTarget(target.id);
     }
 
-    return(
-        <ItemCard
-            title={target?.sprite?.name}
-            selected={editingTarget?.id === target.id}
-            onClick={onClick}
-            key={target?.sprite?.name}
-            width={120}
-            height={120}
-        >
-            <CostumeImage costume={target?.sprite?.costumes[0]} className="costumeCardImage" />
-        </ItemCard>
+    return (
+        <>
+            {target?.sprite?.name.length > 13 ?
```

Comment:
Would probably refactor this ternary into an if statement in the body of the component's function. Just so we don't have to repeat a bunch of JSX